lib/engine/stock_market.rb

```diff
@@ -23,14 +23,15 @@ def set_par(corporation, share_price)
       corporation.par_price = share_price
     end
 
-    def move_right(corporation)
+    def move_right(corporation, jumps)
       r, c = corporation.share_price.coordinates
-
-      if c + 1 < @market[r].size
-        c += 1
-        move(corporation, r, c)
-      else
-        move_up(corporation)
+      jumps.times do
```

Comment:
jumps ||= 1

Reply:
Changed this to       (jumps ||= 1).times do
I don't know if this is correct

Reply:
I couldn't figure out what to do with this so I changed it back to jumps.times.do